deb/build

```diff
@@ -2,6 +2,14 @@
 #
 # script using fpm <https://github.com/jordansissel/fpm> to build libdrizzle-redux
 # runtime, debugging and development deb packages
+#
+# Usage: ./build [api_level]
+#
+#     api_level        the api level for the package
+#
+# The `api_level` will be appended to the `package_name` for the generated
+# packages, i.e. running `./build 6` would generate a packages named
+# `libdrizzle-redux6`, `libdrizzle-redux-dbg`, `libdrizzle-redux-dev`.
```

Comment:
So `libdrizzle-redux6` is the only lib with the suffix? Not `libdrizzle-redux-dbg` and `libdrizzle-redux-dev`?